src/calculateRentalCost.js

```diff
@@ -4,7 +4,23 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
```

Comment:
Consider using `const` instead of `let` for the `calculateRentalCost` function declaration if you do not plan to reassign it, to prevent accidental reassignment and signal that its value should not change.